test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.util.TypicalPersons;
+
+import static org.junit.Assert.*;
```

Comment:
you are encouraged to use explicit imports

Reply:
Noted, and updated! (: